### PR TITLE
Add tweaks, add some kind of parsing for torrents etc.

### PR DESCRIPTION
**Pull request by Gaisberg**

### CODE REVIEW

File: backend/program/scrapers/torrentio.py [hunk 1]
```diff
@@ -20,9 +21,7 @@ def __init__(self, media_items: MediaItemContainer):
         self.class_settings = settings_manager.get(self.settings)
         self.last_scrape = 0
         self.filters = self.class_settings["filter"]
-        self.minute_limiter = RateLimiter(
-            max_calls=60, period=60, raise_on_limit=True
-        )
+        self.minute_limiter = RateLimiter(max_calls=60, period=60, raise_on_limit=True)
```
Comment (dreulavelle) [inline backend/program/scrapers/torrentio.py:24]: Looks like your fixing black's formatting

Comment (Gaisberg) [reply backend/program/scrapers/torrentio.py:24]: Then black formatted its formatting itself, I just ran it again 🗡️

**Comment by dreulavelle** [comment]
fixed conflicts, need to pull

**Review by dreulavelle — APPROVED** [review]
Yolo

**Pull request merged**